pages/about.html

```diff
@@ -4,9 +4,58 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="styles/index.css" rel="stylesheet"/>
   <title>Document</title>
```

Comment:
Make sure to update the title for each page! This is the text that shows up in the tab in the browser.